server/index.js

```diff
@@ -1,11 +1,42 @@
 require('dotenv/config');
+const pg = require('pg');
 const express = require('express');
+const argon2 = require('argon2');
 const staticMiddleware = require('./static-middleware');
 const errorMiddleware = require('./error-middleware');
 
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  // What does the ssl reject unauthorized do?
```

Comment:
If `rejectUnauthorized` is `true` then you must access the database only with an actual, authorized SSL token. (Which is basically `https`, which you can't use on localhost.)